src/component/content.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
+
+class Content extends React.Component{
+    constructor(props){
+        super(props);
+        this.state={
+
+        }
+
+    
+    }
+
+    render(props){
+        return(
+            <div className="container">
+                <div className="row">
+                    <div className="col-md-4"></div>
+                        <h1>hello {this.props.firstname [0]}  {this.props.lastname}</h1>
```

Comment:
extra space and missing indexing on one of them I would suggest releasing this component of the indexing responsibility and instead keep that logic on the parent and only pass strings down to this one via props
```suggestion
                        <h1>hello {this.props.firstname[0]}  {this.props.lastname[0]}</h1>
```